packages/core/src/components/Carousel/Carousel.js

```diff
@@ -1,51 +1,79 @@
+            <Icon className={cx(css.chevron, css.prevIcon)} name="chevron" />
+            <ScreenReadable>{props.accessibilityPrevLabel}</ScreenReadable>
+          </BtnContainer>
+          <BtnContainer
+            onClick={() => setSlideIndex(slideIndex + 1)}
+            className={css.button}
+            disabled={slideIndex === props.children.length - props.slidesToShow}
+          >
+            <Icon className={cx(css.chevron, css.nextIcon)} name="chevron" />
+            <ScreenReadable>{props.accessibilityNextLabel}</ScreenReadable>
+          </BtnContainer>
+        </div>
+      )}
+    </div>
+  );
+
+  const renderCenterLeftControls = () => {
+    if (!props.infinite && slideIndex === 0) return null;
+    return (
+      <BtnContainer onClick={() => setSlideIndex(slideIndex - 1)} className={cx(css.control, css.prev)}>
+        <Icon className={css.prevIcon} name="chevron" />
+        <ScreenReadable>{props.accessibilityPrevLabel}</ScreenReadable>
+      </BtnContainer>
+    )
+  }
+
+  const renderCenterRightControls = () => {
+    if (!props.infinite && slideIndex === props.children.length - props.slidesToShow) return null;
     return (
-      <div className={css.controls}>
-        <SectionHeader title={props.title} level={2} className={css.title} />
-        {props.slidesToShow < props.children.length && (
-          <div className={css.buttons}>
-            <button
-              className={cx(css.button)}
-              disabled={slideIndex === 0}
-              onClick={() => setSlideIndex(slideIndex - 1)}
-            >
-              <Icon className={cx(css.chevron, css.prevIcon)} name="chevron-right" />
-            </button>
-            <button
-              className={cx(css.button)}
-              disabled={slideIndex === props.children.length - props.slidesToShow}
-              onClick={() => setSlideIndex(slideIndex + 1)}
-            >
-              <Icon className={cx(css.chevron, css.nextIcon)} name="chevron-right" />
-            </button>
-          </div>
-        )}
-      </div>
-    );
-  };
+      <BtnContainer onClick={() => setSlideIndex(slideIndex + 1)} className={cx(css.control, css.next)}>
+        <Icon className={css.nextIcon} name="chevron" />
+        <ScreenReadable>{props.accessibilityNextLabel}</ScreenReadable>
+      </BtnContainer>
+    )
+  }
 
   return (
-    <div>
-      {renderDesktopControls()}
+    <div className={css.carousel}>
+      {props.useTopRightControls && renderTopRightControls()}
+      {props.useCenterControls && renderCenterLeftControls()}
+      {props.useCenterControls && renderCenterRightControls()}
       <NukaCarousel slideIndex={slideIndex} {...props}>
-        {props.children.map((slide) => (
-          <div key={shortid.generate()} className={css.slideInner}>
-            {slide}
-          </div>
-        ))}
+        {props.children.map((slide) => {
+          const id = nanoid();
+          return (
+            <div key={id} className={css.slideInner}>
```

Comment:
the declaration of id is redundant:
```suggestion
            <div key={nanoid()} className={css.slideInner}>
```